src/server/common/model/answer/origin-type/origin-type.js

```diff
@@ -19,12 +20,28 @@ const afuOption = {
   hint: 'Including enhanced with grazing (AFUE)'
 }
 
-const offFarmOptions = {
+const offFarmOptions = () =>
```

Comment:
if moved down to below the declarations, this wouldnt need to be a function meaning the ternary on line 65 would look similar for both conditions (ie neither is a function call) - this would maintain the same composition and reduce call stack

Reply:
@DrogoNevets  - it's in here so that it's evaluated late so we can test it with `spyOnConfig`